modules/cudacodec/include/opencv2/cudacodec.hpp

```diff
@@ -329,6 +329,7 @@ struct CV_EXPORTS_W_SIMPLE FormatInfo
     CV_PROP_RW cv::Size targetSz;//!< Post-processed size of the output frame.
     CV_PROP_RW cv::Rect srcRoi;//!< Region of interest decoded from video source.
     CV_PROP_RW cv::Rect targetRoi;//!< Region of interest in the output frame containing the decoded frame.
+    CV_PROP_RW bool videoFullRangeFlag;//!< indicates if the black level, luma and chroma are using the full range 0-255.
```

Comment:
The flag corresponds to BGR 709 HDTV (true) / RGB 709 CSC (false) conversion. I propose to use it naming or at least to define it in documentation string. Also, it makes sense to mention, that the flag is set automatically from stream parser.

Reply:
> The flag corresponds to BGR 709 HDTV (true) / RGB 709 CSC (false) conversion. I propose to use it naming or at least to define it in documentation string. 

The flag indicates whether the luma and chroma samples are represented using the full range of 8 bit values (0-255) or a smaller range.  From my understanding the full range is usually but not limited to use for TV sources with the smaller range being the most common for video.  This is applicable to BGRA, BGR and Gray conversion not just BGRA.

The name is to be consistent with the definition in [Annex E of the ITU-T Specification](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&ved=2ahUKEwiD776-u6b-AhWyX_EDHc5LBc4QFnoECAoQAQ&url=https%3A%2F%2Fwww.itu.int%2Frec%2Fdologin_pub.asp%3Flang%3De%26id%3DT-REC-H.264-201602-S!!PDF-E%26type%3Ditems&usg=AOvVaw3vL-wKcGo4yatSlDHUtC5q) (video_full_range_flag) and the name given internally in the Nvidia Video Codec SDK (videoFullRangeFlag).  I can add these details to the documentation string but I think the name is valid?

> Also, it makes sense to mention, that the flag is set automatically from stream parser.

Agreed.

Reply:
Ok, let's presume naming, but add reference to BGR 709 HDTV / RGB 709 color spaces.